source/common/network/utility.cc

```diff
@@ -190,13 +190,25 @@ void Utility::throwWithMalformedIp(absl::string_view ip_address) {
   throw EnvoyException(absl::StrCat("malformed IP address: ", ip_address));
 }
 
+#define SUPPORTS_GETIFADDRS
+#ifdef WIN32
+#undef SUPPORTS_GETIFADDRS
+#endif
+
+// https://android.googlesource.com/platform/prebuilts/ndk/+/dev/platform/sysroot/usr/include/ifaddrs.h
+#ifdef __ANDROID_API__
+#if __ANDROID_API__ < 24
+#undef SUPPORTS_GETIFADDRS
+#endif // __ANDROID_API__ < 24
+#endif // ifdef __ANDROID_API__
```

Comment:
Move this part to https://github.com/envoyproxy/envoy/blob/master/include/envoy/common/platform.h ?

Reply:
moved